### DISCUSSION TITLE
Push video quality settings to device.

### DISCUSSION THREAD

**Discussion by dem0278** [AxisCommunications/discussions#86] [VAPIX] [answered]
Going over the vapix library, I'm seeing commands to request streams of specific quality or confirm the current quality settings on the device. Hoping someone can point me to documentation detailing how to push video quality settings to the device. For example, I have coworkers who raise the framerate on cameras when investigating a motion detection alarm and sometimes forget to lower the fps back down to our preferred setting. For cameras where bandwidth usage is a real concern, I would appreciate the ability to send a vapix command that ensures the fps and/or resolution is set per our requirements.

**Comment by mattias-kindborg-at-work** [comment]
Hi @dem0278! Axis cameras support the concept of stream profiles, describer in the VAPIX documentation [here](https://www.axis.com/vapix-library/subjects/t10175981/section/t10149933/display). I'm guessing that this is what you are looking for?

**Reply by dem0278** [reply]
Hi @mattias-kindborg-at-work, Thanks for the suggestion. Unfortunately, pushing or altering a stream profile is not quite what I'm after. Looking to push video quality settings to the device in a way that changes its web interface live stream parameters, if possible.

**Comment by dem0278** [comment, marked as answer]
Got it figured out. 
http://"servername"//axis-cgi/param.cgi?action=update&root.image.I0.stream.fps="value"
Thanks!
